pipeline/matcher/src/main/scala/weco/pipeline/matcher/matcher/WorkMatcher.scala

```diff
@@ -47,11 +47,12 @@ class WorkMatcher(
         // but I haven't thought hard enough about whether it might introduce a
         // hard-to-debug consistency error if another process updates the graph
         // between us reading it and writing it.
+        result = MatcherResult(
+          works = toMatchedIdentifiers(afterNodes),
+          createdTime = Instant.now())
```

Comment:
Is there potentially a reason these were created separately? I ask as it looks like createdTime _might_ be different?

Reply:
You're right, the createdTime on the latter will be after the successful write to the store.